prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1380,6 +1457,7 @@ void rxr_pkt_handle_eager_rtw_recv(struct rxr_ep *ep,
 		return;
 	}
 
+	ofi_copy_to_iov(rx_entry->iov, rx_entry->iov_count, 0, data, data_size);
```

Comment:
Shouldn't this still update `rx_entry->bytes_done`? Same thing for the long rtw message.